src/modules/Topics/utils/utils.ts

```diff
@@ -72,8 +72,8 @@ type ConversionUnit = {
 
 export const millisecondsToTime = (value: number): ConversionUnit => {
   if (value) {
-    if (value % unitsToMilliSecond.day == 0)
-      return { value: value / unitsToMilliSecond.day, unit: 'days' };
+    if (value % unitsToMilliSecond.days == 0)
+      return { value: value / unitsToMilliSecond.days, unit: 'days' };
     if (value % unitsToMilliSecond.hours == 0)
```

Comment:
I think `days` etc. should also use constants. It wasn't initially clear that those are correlated.